src/statefulmodelinstance.cpp

```diff
@@ -27,6 +27,10 @@ using namespace InferenceEngine;
 
 namespace ovms {
 
+StatefulModelInstance::~StatefulModelInstance() {
+    sequenceManager->join();
+}
```

Comment:
Do we need this? Sequence manager destructor will  be called along with this one and will also call `join()` method